syntax = "proto3";

import "google/protobuf/empty.proto";

message Metadata {
    string name = 1;
    string firmwareVersion = 2;
    string birthday = 3;
    string serialID = 4;
    string batteryType = 5;
}

service MetaService {
    rpc GetMetadata (google.protobuf.Empty) returns (Metadata) {}
}